### PR TITLE
feat: asset optimization

### PR DESCRIPTION
**Pull request by AlexanderDevitsky**
## Related Issue

https://github.com/vuestorefront/magento2/issues/397
https://vsf.atlassian.net/browse/M2-45

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/components/AppHeader.vue [hunk 1]
```diff
@@ -122,29 +115,29 @@
...
+                height="18"
+                class="sf-search-bar__icon"
+              />
             </SfButton>
             <SfButton
               v-else
               class="sf-search-bar__button sf-button--pure"
               aria-label="Open search"
-              @click="isSearchOpen ? isSearchOpen = false : isSearchOpen = true"
+              @click="isSearchOpen ? isSearchOpenx = false : isSearchOpen = true"
```
Comment (Frodigo) [inline packages/theme/components/AppHeader.vue:130]: `isSearchOpenx` Looks like a typo mistake

File: packages/theme/components/General/IconSprite.vue [hunk 2]
```diff
@@ -0,0 +1,362 @@
+<template>
+  <svg width="0" height="0" style="position: absolute; visibility:hidden">
+    <symbol xmlns="http://www.w3.org/2000/svg" viewBox="0 0 231 221.4210525" id="logo">
```
Comment (Frodigo) [inline packages/theme/components/General/IconSprite.vue:3]: Today we updated the logo, please double-check if this is a new one https://github.com/vuestorefront/magento2/pull/622

**Review by Frodigo — CHANGES_REQUESTED** [review]

File: packages/theme/components/AppHeader.vue (hunk 1 above)
Comment (AlexanderDevitsky) [reply packages/theme/components/AppHeader.vue]: @Frodigo Fixed

**Review by Frodigo — APPROVED** [review]
@AlexanderDevitsky looks good to me. Please resolve conflicts, then I will merge it and we will be able to test your solution on our servers. 

Thanks,
Marcin

**Comment by AlexanderDevitsky** [comment]
@Frodigo resolved

**Pull request merged**